### ISSUE TITLE
Unable to edit boolean values v0.3.19

### ISSUE THREAD

**Issue by bscheibe** [keystonejs/keystone#3050]
I recently updated from version `v0.3.12` to `v0.3.19` and am now unable to edit boolean values through the admin UI. The fields are all set and functioning in the DB, and I've changed all my `Boolean` to `Types.Boolean` in the schema just to be safe.

When I try to click on one of the values I receive this error in the console.`Uncaught TypeError: Cannot read property 'length' of undefined` for `keystone/packages.js`. The checkbox does not change. 

![screen shot 2016-06-17 at 8 53 05 pm](https://cloud.githubusercontent.com/assets/2762046/16162876/04ed4ace-34ce-11e6-9484-1f4040d7843b.png)

Did I screw up the update somewhere or is this a bug? I'll close this issue asap if you believe it not to be a bug.

**Comment by creynders** [comment]
Could you update to v0.3.21 and check whether the issue still exists? In the last two releases some changes were made to the boolean field.

**Comment by bscheibe** [comment]
Updated to `v0.3.21` and the error is still occurring. Thanks for the attempted solution.

![screen shot 2016-06-21 at 8 38 47 pm](https://cloud.githubusercontent.com/assets/2762046/16243648/393baa30-37f0-11e6-9a37-255431ed763c.png)

**Comment by bscheibe** [comment]
So I did a little line debugging. I expanded `package.js` and found that for some reason when I click on the Boolean value the event that actually appears is the a `Search for an image from Cloudinary event...` whose value is undefined.

It seems to swallow all the events of Boolean values on the page (when they're clicked). So instead of receiving the Boolean change event it receives the Cloudinary `field_image_select` object.

![screen shot 2016-06-24 at 6 36 09 pm](https://cloud.githubusercontent.com/assets/2762046/16345610/b55a7424-3a3a-11e6-9696-2c579cf93619.png)

I can do without the select an image field. Which if I remove everything works 👌 

However I can continue to do more digging if necessary, just let me know. However I thought I should post this summary so far.
